src/Workspaces/Core/Portable/Shared/Extensions/ICodeDefinitionFactoryExtensions.cs

```diff
@@ -24,6 +34,17 @@ internal static partial class ICodeDefinitionFactoryExtensions
                     SpecializedCollections.EmptyList<SyntaxNode>()));
         }
 
+        private static SyntaxNode CreateQualifiedThrowNotImplementStatement(
+            this SyntaxGenerator codeDefinitionFactory,
+            Compilation compilation)
+        {
+            return codeDefinitionFactory.ThrowStatement(
```

Comment:
This is not the right way to do this generation.  It won't properly qualify wiht htings like global:: when necessary.  

The better fix would be to set the code generation option to not add usings.